packages/trader/src/AppV2/Components/StreamIframe/stream-iframe.tsx

```diff
@@ -34,7 +39,8 @@ const StreamIframe = ({
     ].join('&');
 
     return (
-        <div className='stream__wrapper'>
+        <div className={clsx('stream__wrapper', is_loading && 'stream__wrapper--is-loading')}>
+            {is_loading && <Skeleton.Square height={`calc(100vw * ${ASPECT_RATIO})`} />}
```

Comment:
Why we are passing height like this (not via css className)?
Because in Quill default height is 100% and it will be insert inline. If we pass height in className it will be rewritten by default height value.
<img width="522" alt="Screenshot 2024-10-21 at 4 36 04 PM" src="https://github.com/user-attachments/assets/a7efd1e6-c9e7-426f-837a-94a67c3a6ec0">